script/github-actions/select-cypress-tests.js

```diff
@@ -129,9 +129,12 @@ function selectedTests(graph, pathsOfChangedFiles) {
     .map(filePath => filePath.split('/')[2]);
 
   [...new Set(applicationNames)].forEach(app => {
-    // Lookup app in cross-app imports graph to reference which app's tests
-    // should run
-    applications.push(...graph[app].appsToTest);
+    // Bypass if app has been from directory
+    if (graph[app]) {
```

Comment:
This git action was failing due to `graph[app]` being `undefined`. This was happening since `graph = current directory application` and `app = deleted application`.

This is just a quick check to ensure the changed application was not deleted before adding it to the `tests`